ui/packages/consul-ui/translations/en-us.yaml

```diff
@@ -184,6 +184,46 @@ components:
             name: Precedence
             asc: Ascending
             desc: Descending
+    transparent-proxy: Transparent Proxy
```

Comment:
@johncowen this file is getting really big now. 😬

Reply:
Yeah ideally the translations for components needs to be split up to live in the components folders with all the rest of the component code. There are other things we can do to split it up also, is that something you can look at?

Reply:
I tried splitting things into folder translation/components/transparent-proxy but it didn't work. Do I have to add the new files in settings?

Reply:
Oh sorry, we'd have to do some broccoli magic to make the component folder thing work, there are other ways to split it up though, take a look at the docs there's some guidance there.